create searchResultsPage

GitJustinLau opened the pull request:
add grid and list icons
take AddToCartButton out of folder
adjust header styling
adjust SearchBar styling

GitJustinLau committed:
create searchResultsPage
add grid and list icons
take AddToCartButton out of folder
adjust header styling
adjust SearchBar styling

GitJustinLau committed:
fix routing to searchresults page

koo-mi commented:
Nice job creating the page and making adjustments to other components too! 

You are free to merge this but there are few minor things that I would like to suggest.
- use main tag for the page instead of react fragment
- use different background color for the Filter / Best Match
- remove {' '} in SearchResultPage. It seems like prettier sometimes create those.
- center the alignment of the hamburger menu icon in Header component

GitJustinLau committed:
align hamburger menu in header
change color of filter subheader on search results page
remove {' '} from code
replace fragments with <main/>

GitJustinLau committed:
remove gap between filter and best match

GitJustinLau committed:
remov gap from section element
re-add gap to button grouping ( filter/best match)
change the section bg color to 200 and buttons to 300

koo-mi commented:
Nice job fixing it! I will merge it.
